### ISSUE TITLE
Problem with Kinesis Advanced 360 Pro

### ISSUE THREAD

**Issue by Manubi** [KinesisCorporation/Adv360-Pro-ZMK#347]
I wanted to migrate from V2.0 to V3.0 of the firmware.

I run on an apple macbook pro intel 2019 macOS Sonoma 14.2.1
[config repo](https://github.com/Manubi/Adv360-Pro-ZMK/)

**Problem 1**: when i put the keyboard in bootloader mode (3 green lights) and i just drag the .uf2 file from the finder onto the volume ADV360PRO i get the following error:
![CleanShot 2024-01-19 at 14 35 08@2x](https://github.com/KinesisCorporation/Adv360-Pro-ZMK/assets/20935138/339bc837-57a1-4273-a6f5-7706501986d7)

After a while of googling I was able to fix it with simply copying with 'cp left.uf2 /Volumes/ADV360PRO' which did work till now.

**Problem 2:** Now if I want to update the keyboard in the terminal with the cp command I get the following error:

![CleanShot 2024-01-19 at 14 35 28@2x](https://github.com/KinesisCorporation/Adv360-Pro-ZMK/assets/20935138/4b8757e2-0db1-4740-937c-235598efb43e)

The keyboard goes into the copy mode... blinks gives me the above error and jumps back into bootloader mode (3 green lights).

The only reason I wanted to try to update to firmware V3 was to try again if the bluetooth would work with my mac...

Has anyone an idea how i an fix this mess?

Thanks

**Comment by ReFil** [comment]
The board jumping back into bootloader mode after flashing the settings reset file is expected behaviour.

The finder error is not indicative of problems. Please see this note from ZMK on the topic: https://zmk.dev/docs/troubleshooting#file-transfer-error

**Comment by Manubi** [comment]
I downgraded back to V2.0 and now finally everything is working again including the bluetooth connection. I guess there is no real benefit in updating if everything is working fine?

I would love to see the documentation a little bit improved. Meaning that getting error messages is the 'normal' as the happy path. Because I've just forgotten that it's supposed to work like this. Would have saved me a couple hours of trying to figure out how I did it almost 2 years ago.
